Assets/Scripts/Actor Components/Health.cs

```diff
@@ -1,16 +1,30 @@
 using System.Collections;
 using System.Collections.Generic;
 using UnityEngine;
+using UnityEngine.Events;
+using Photon.Pun;
+using Photon.Realtime;
 
 public class Health : MonoBehaviour
 {
     [SerializeField] private int healthPoints;
+    [SerializeField] private UnityEvent decrementHealthEvent;
 
     public int HealthPoints { get => healthPoints; }
 
+    public UnityEvent DecrementHealthEvent { get => decrementHealthEvent; }
+
     public void Decrement()
+    {
+        PhotonView photonView = GetComponent<PhotonView>();
```

Comment:
Instead of using `GetComponent` every time health decreases, consider creating a serialized `photonView` field above and setting it in the editor (since all the actors' prefabs have the photon view component already). `GetComponent` can be costly, so we try to avoid it if possible.